protocols/luci-proto-batman-adv/Makefile

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (C) 2008-2014 The LuCI Team <luci@lists.subsignal.org>
```

Comment:
You should put your own copyright there. Have a look at the other protocols in this directory.

Reply:
OK - will do, thanks!